docs/rules/require-expect.md

```diff
@@ -28,6 +40,28 @@ The "never-except-zero" option disallows `except` calls, except when used to
 explicitly assert that a test performs no assertions, which would otherwise
 be considered an error.
 
+### Config Options
+
+This rule allows exactly one config option: `allowFunctions`
+
+#### allowFunctions
+
+If you are using the Check Type `except-simple` whenever you pass `assert` into
+another function you will cause an error. If you have some functions that you
+are passing `assert` to that don't add to the number of asserts (and maybe just
+extract the test name from the assert context) then you can mark those functions
+to be ok with the `allowFunctions` config option.
+
+Here is an example:
+
+```js
+rules: {
```

Comment:
In order for eslint-plugin-markdown to pass, please surround this object with curly braces and remove the dangling comma at the end. Thanks!